src/stores/session.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from 'pinia';
```

Comment:
todo: please also update this file according to our common project from the last year to get rid of the unnecessary types.